integrations/datadog/.port/resources/port-app-config.yaml

```diff
@@ -35,9 +35,8 @@ resources:
             tags: .attributes.schema.tags
             type: .attributes.schema.type
             links: .attributes.schema.links | map(.url)
-            email: '.attributes.schema.contacts[] | select(.type == "email") | .contact'
-            slack: '.attributes.schema.contacts[] | select(.type == "slack") | .contact'
-            teams: '.attributes.schema.contacts[] | select(.type == "microsoft-teams") | .contact'
+            owners: '[.attributes.schema.contacts[] | select(.type == "email") | .contact]'
+            slack: '[.attributes.schema.contacts[] | select(.type == "slack") | .contact]'
```

Comment:
remove slack

Reply:
it is the slack url of the service owners